src/GitInfo/build/GitInfo.targets

```diff
@@ -50,6 +50,14 @@
 					 in a previous build in a GitInfo.cache for 
 					 performance reasons. 
 					 Defaults to empty value (no ignoring).
+           
+  $(GitTagRegex): Regular Experssion used with git describe to find the BaseTag

```

Comment:
I don't think this is really needed. We could just run the version regex against the tags in order to filter out the ones to consider

Reply:
I think you are right, but I didn't have an easy way to test, so I kept them separate to make clear what I thought would help.